README.md

```diff
@@ -150,6 +155,32 @@ The custom configuration options for the REST API are listed below:
   [Flask-SQLAlchemy configuration](https://flask-sqlalchemy.palletsprojects.com/en/2.x/config/#configuration-keys)
   documentation.
 
+The custom configuration options for AMQP 1.0 messaging are listed below:
+
+* `IIB_MESSAGING_BATCH_STATE_DESTINATION` - the AMQP 1.0 destination to send the batch state change
+  messages. If this is not set, IIB will not send this type of message. If this is set,
+  `IIB_MESSAGING_URLS` must also be set.
+* `IIB_MESSAGING_BUILD_STATE_DESTINATION` - the AMQP 1.0 destination to send the build request state
+  change messages. If this is not set, IIB will not send this type of message. If this is set,
+  `IIB_MESSAGING_URLS` must also be set.
+* `IIB_MESSAGING_CA` - the path to a file with the certificate authority that signed the certificate
+  of the AMQP 1.0 message broker. This defaults to `/etc/pki/tls/certs/ca-bundle.crt`.
+* `IIB_MESSAGING_CERT` - the path to the identity certificate used for authentication with the
+  AMQP 1.0 message broker. This defaults to `/etc/iib/messaging.crt`.
+* `IIB_MESSAGING_DURABLE` - determines if the messages are durable and cannot be lost due to an
```

Comment:
@lcarva this is different. This is in regards to the meaning of durable according to the AMQP 1.0 spec and not an ActiveMQ thing.

Here is the snippet:

> Durable messages MUST NOT be lost even if an intermediary is unexpectedly terminated and restarted. A target which is not capable of fulfilling this guarantee MUST NOT accept messages where the durable header is set to true: if the source allows the rejected outcome then the message SHOULD be rejected with the precondition-failed error, otherwise the link MUST be detached by the receiver with the same error.

http://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-complete-v1.0-os.pdf

Reply:
ah ok, and the target here is the broker, not the consumer, right?

Reply:
Correct.